clusters/hive/pools/openshift-ci/ci-ocp-4-12-0-amd64-aws-us-east-1_clusterpool.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: hive.openshift.io/v1
+kind: ClusterPool
+metadata:
+  creationTimestamp: null
+  labels:
+    architecture: amd64
+    cloud: aws
+    owner: openshift-ci
+    product: ocp
+    region: us-east-1
+    version: "4.12"
+    version_lower: 4.12.0-0
+    version_upper: 4.13.0-0
+  name: ci-ocp-4-12-amd64-aws-us-east-1
+  namespace: ci-cluster-pool
+spec:
+  baseDomain: hive.aws.ci.openshift.org
+  hibernationConfig:
+    resumeTimeout: 20m0s
+  imageSetRef:
+    name: ocp-release-4.12.1-x86-64-for-4.12.0-0-to-4.13.0-0
```

Comment:
I see `ocp-release-4.12.1-x86-64-for-4.12.0-0-to-4.13.0-0` configured nowhere as of now. Do we have to wait for `clusterimageset-updater` to sort things out, right?

Reply:
I believe we have automation to refresh it.

Reply:
Yeah, just wanted to be sure and have your opinion.